device_backends/Subdevice/src/SubdeviceBackend.cc

```diff
@@ -351,7 +343,7 @@ namespace ChimeraTK {
   template<typename UserType>
   boost::shared_ptr<NDRegisterAccessor<UserType>> SubdeviceBackend::getRegisterAccessor_area(
       const RegisterPath& registerPathName, size_t numberOfWords, size_t wordOffsetInRegister, AccessModeFlags flags) {
-    assert(type == Type::area);
+    assert(hasAreaParam());
```

Comment:
I think this must stay at the old assert, since this case is only for the area type and not for the areaHandshake, right?

Reply:
yes